packages/fluent/scss/bottom-navigation/_variables.scss

```diff
@@ -41,7 +45,7 @@ $kendo-bottom-nav-item-min-width: 72px !default;
 $kendo-bottom-nav-item-max-width: none !default;
 /// Minimum height of the bottom navigation item.
 /// @group bottom-navigation
-$kendo-bottom-nav-item-min-height: calc( var( --kendo-icon-size, 1rem ) + #{$kendo-padding-x * 2} - #{$kendo-bottom-nav-padding-x * 2} ) !default;
```

Comment:
the min-height should be related to the icon size. I would suggest replacing kendo-spacing 6 with $kendo-bottom-nav-item-icon-size var that you have defined above